server/sql/analysis/clck_view.sql

```diff
@@ -0,0 +1,22 @@
+--
+-- View for Intel(R) Cluster Checker
+--
+DROP VIEW IF EXISTS clck_1;
+CREATE OR REPLACE VIEW clck_1 AS
+    SELECT test_run.duration,
+        0 AS encoding,
+        test_run.exit_value AS exit_status,
+        submit.hostname,
+        ''::text AS node_names,
+        0 AS num_nodes,
+        test_run.test_run_command_id::character varying AS optionid,
+        test_run.ctid AS rowid,
```

Comment:
ctid is a system column we used as a unique row identifier to test with CLCK. You can find the definition of ctid at [this](https://www.postgresql.org/docs/9.5/static/ddl-system-columns.html) link. However, after discussing with the team, we will use the existing test_run_id column as a unique row identifier. I will update the view.